.travis/build-pkgs

```diff
@@ -65,11 +66,11 @@ done
 
 printf '\n----\n'
 if [ -n "$successful_pkgs" ]; then
-	print -s1 -c2 "Successfully build packages:$successful_pkgs\n"
+	print -s1 -c2 "Successfully build packages: $successful_pkgs\n"
 fi
 if [ -n "$failed_pkgs" ]; then
-	die "Failed to build packages:$failed_pkgs"
+	die "Failed to build packages: $failed_pkgs"
 
 elif [ -z "$successful_pkgs" ]; then
```

Comment:
Since this is an `elif`, it will NOT execute the warning `if [ -n "$failed_pkgs" ]`. So if we go to this block, we know that:
- there are no failed packages.
- there are no successfully built packages.

Therefore, no package build was triggered.